pootle/core/forms.py

```diff
@@ -190,7 +190,8 @@ class FormWithActionsMixin(forms.Form):
     select_all = forms.BooleanField(
         required=False,
         label=_(
-            "Select all matching filter criteria, including those not shown"),
+            "Select all items matching filter criteria, including those not "
+            "shown"),
```

Comment:
Given the actual usage I can't tell where this is used for, so will keep "items".